internal/binutils/binutils_test.go

```diff
@@ -188,15 +190,51 @@ func skipUnlessDarwinAmd64(t *testing.T) {
 	}
 }
 
+// ensureObjdump checks if a supported version of objdump is availabe. Keep the test simple
+// and check for LLVM objdump v11.0+ or a GNU Binutils objdump.
+func ensureObjdump() bool {
+	cmdOut, _ := exec.Command("objdump", "--version").Output()
+	// Keep this regexp in sync with the one used in binutils.go
+	//	re := regexp.MustCompile(`LLVM\sversion (([0-9]*)\.([0-9]*)\.([0-9]*)).*\s`)
```

Comment:
This line and the comment above can be removed, as the code reuses the regular expression so there is nothing to keep in sync?

Reply:
Forgot to remove. Removed now